tests/store.rs

```diff
@@ -0,0 +1,54 @@
+#![expect(clippy::panic_in_result_fn, reason = "Panics OK in tests.")]
+
+use core::error::Error;
+pub mod fixture;
+use fixture::{add_pod_storage, pod_style, store_test};
+use orcapod::{
+    error::FileHasNoParent,
+    model::{to_yaml, Pod},
+};
+use std::{fs, path::Path};
+use tempfile::tempdir;
+
+fn is_dir_two_levels_up_empty(file: &Path) -> Result<bool, Box<dyn Error>> {
+    Ok(file
+        .parent()
+        .ok_or_else(|| FileHasNoParent {
+            path: file.to_path_buf(),
+        })?
+        .parent()
+        .ok_or_else(|| FileHasNoParent {
+            path: file.to_path_buf(),
+        })?
+        .read_dir()?
+        .next()
+        .is_none())
+}
+
+#[test]
+fn verify_pod_save_and_delete() -> Result<(), Box<dyn Error>> {
+    let store_directory = String::from(tempdir()?.path().to_string_lossy());
+    {
+        let pod_style = pod_style()?;
+        let store = store_test(Some(&store_directory))?; // new tests can just call store_test(None)?
+        let annotation_file = store.make_annotation_path(
+            "pod",
+            &pod_style.hash,
+            &pod_style.annotation.name,
+            &pod_style.annotation.version,
+        );
+        let spec_file = store.make_spec_path("pod", &pod_style.hash);
+        {
+            let pod = add_pod_storage(pod_style, &store)?;
+            assert!(fs::exists(&spec_file)?);
```

Comment:
```suggestion
            assert!(spec_file.exists());
```

Using the pathbuf .exist() function might make it look cleaner: https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.exists